scripts/talos/bumpSimorghPackages/getUpdates/index.js

```diff
@@ -0,0 +1,13 @@
+const getUpdates = (dependencies, publishedPackages) => {
+  const updates = [];
+  Object.keys(publishedPackages).forEach(packageName => {
+    const version = dependencies[packageName];
+    if (version)
+      updates.push(
```

Comment:
No big deal but I try to discourage the use of mutation where possible. I think `map` and `filter(Boolean)` would be cleaner than mutating a variable outside the scope of a `forEach`

Reply:
Yeah, initially updates needed to be an object thus the mutation. I'll refactor to array functions now.